dataform/examples/dataform_udf_unit_test/definitions/test_cases.js

```diff
@@ -0,0 +1 @@
+../../../../udfs/community/test_cases.js
```

Comment:
Is this supposed to be a symlink? Not sure I understand the contents of this file.

Reply:
yep it's a symlink, the example runs the unit tests for the bqutil community UDFs